features/steps/steps.py

```diff
@@ -1157,6 +1157,20 @@ def when_i_create_local_ppas(context, release, next_release):
     )
 
 
+@when("I verify root and non-root `{cmd}` calls have the same output")
+def status_root_vs_nonroot_comparison(context, cmd):
```

Comment:
I'd just rename the function (and variables) as the functionality isn`t only related to status - can be used with any command actually.